src/mpyl/cli/commands/build/mpyl.py

```diff
@@ -45,40 +46,70 @@ class MpylRunParameters:
 FORMAT = "%(name)s  %(message)s"
 
 
-def get_build_plan(logger: logging.Logger, repo: Repository, mpyl_run_parameters: MpylRunParameters) -> RunResult:
+def get_build_plan(
+    logger: logging.Logger, repo: Repository, mpyl_run_parameters: MpylRunParameters
+) -> RunResult:
     params = mpyl_run_parameters.parameters
-    branch = repo.get_branch or mpyl_run_parameters.run_config.run_properties.versioning.branch
+    branch = (
+        repo.get_branch
+        or mpyl_run_parameters.run_config.run_properties.versioning.branch
+    )
     logger.info(f"Running with {params}")
     if branch:
         if repo.main_branch_pulled:
-            logger.info(f'Branch `{repo.main_branch}` already present locally. Skipping pull.')
+            logger.info(
+                f"Branch `{repo.main_branch}` already present locally. Skipping pull."
+            )
         else:
-            logger.info(f'Pulling `{repo.main_branch}` from {repo.get_remote_url}')
+            logger.info(f"Pulling `{repo.main_branch}` from {repo.get_remote_url}")
             pull_result = repo.pull_main_branch()
-            logger.info(f'Pulled `{pull_result[0].remote_ref_path.strip()}` to local')
-        changes = repo.changes_in_branch_including_local() if params.local else repo.changes_in_branch()
+            logger.info(f"Pulled `{pull_result[0].remote_ref_path.strip()}` to local")
+        changes = (
+            repo.changes_in_branch_including_local()
+            if params.local
+            else repo.changes_in_branch()
+        )
     else:
-        changes = repo.changes_in_tagged_commit(params.tag) if params.tag else repo.changes_in_merge_commit()
-    logger.debug(f'Changes: {changes}')
-
-    projects_per_stage: dict[Stage, set[Project]] = find_build_set(repo, changes, params.all)
-    return RunResult(run_properties=mpyl_run_parameters.run_config.run_properties, run_plan=projects_per_stage)
+        changes = (
+            repo.changes_in_tagged_commit(params.tag)
+            if params.tag
+            else repo.changes_in_merge_commit()
+        )
+    logger.debug(f"Changes: {changes}")
+
+    projects_per_stage: dict[Stage, set[Project]] = find_build_set(
+        repo, changes, params.all
+    )
+    return RunResult(
+        run_properties=mpyl_run_parameters.run_config.run_properties,
+        run_plan=projects_per_stage,
+    )
```

Comment:
Where are all the formatting changes coming from? Is it from black? Shouldn't those have been implemented already in that case, makes it hard to review the real changes

Reply:
Not sure, but I just rebased on master and the changes are still there.. Looks to me like those files are not formatted on master by black

Reply:
If I use pycharm and black formatter, make some dummy changes in the file and save it it reformats it with black correctly (these changes above). If I do `black .` again, it still reports no changes. So I think something is off with running black in the root, and we should use the IDE to format the code correctly.

Reply:
Fixed in https://github.com/Vandebron/mpyl/pull/158